test/data/testGeoOpt.py

```diff
@@ -123,6 +123,18 @@ def testgeovalues_atomcoords(self):
         msg = "len(atomcoords) is %d but len(geovalues) is %d" % (count_coords, count_geovalues)
         self.assertEquals(count_geovalues, count_coords, msg)
 
+    @skipForParser("GAMESSUK", "Not implemented.")

```

Comment:
So, do we need this test given we do an equivalent test in `testSP`? Are the masses formatted any differently for geometry optimization?

Also, please take the time time to enable this test for ORCA in `testSP`. Thanks!